docker_config/dockerfiles/Dockerfile.server

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
 MAINTAINER CodaLab Worksheets <codalab.worksheets@gmail.com>
 
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && \
   apt-get install -y --no-install-recommends software-properties-common && \
-  add-apt-repository ppa:fkrull/deadsnakes && \
+  add-apt-repository ppa:deadsnakes/ppa && \
```

Comment:
This is great, thanks! This does something similar to https://github.com/codalab/codalab-worksheets/pull/3954 but I forgot to update these dockerfiles in that PR.